sleap_io/model/skeleton.py

```diff
@@ -37,6 +45,13 @@ class Edge:
     source: Node
     destination: Node
 
+    @staticmethod
+    def from_names(edge_list: List[Tuple[str, str]]) -> List["Edge"]:
+        edges = list()
+        for edge in edge_list:
+            edges.append(Edge(source=Node(edge[0]), destination=Node(edge[1])))
+        return edges
+
```

Comment:
Remove this - it's back from the dead (no changes should be made to `models` directory for this PR)